testing/regress/ecl/persist_refresh.ecl

```diff
@@ -0,0 +1,67 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+import Std;
+import Std.File AS FileServices;
+
+//
+//version persistRefresh=true
+//version persistRefresh=false
+
+import ^ as root;
+
+string OriginalTextFilesIp := '.' : STORED('OriginalTextFilesIp');
+string OriginalTextFilesEclPath := '' : STORED('OriginalTextFilesEclPath');
+string OriginalTextFilesOsPath := '' : STORED('OriginalTextFilesOsPath');
+
+persistRefresh := #IFDEFINED(root.persistRefresh, true);
+
+#IF (persistRefresh=true)
+InputFile := OriginalTextFilesOsPath + '/download/europe1.txt';
+#ELSE
+InputFile := OriginalTextFilesOsPath + '/download/europe2.txt';
+#END
```

Comment:
...or maybe we need two test files with both? I feel unsure, considering how many times I hear that persists cause issues with graph flow, eval, & exec etc.